.travis.yml

```diff
@@ -1,14 +1,24 @@
 language: dart
 sudo: required
+
 addons:
   chrome: stable
 
-dart:
-  - 1.24.3
-  - stable
-
-script:
-  - tool/format.sh
-  - dartanalyzer lib bin test/unit
-  - pub publish --dry-run
-  - xvfb-run --server-args="-screen 0 1024x768x24" tool/test.sh
+jobs:
+  include:
+    - dart: 2.4.0
```

Comment:
Same thing - wanna use dart 2.6.1 here?

Reply:
we already have `stable` that will run 2.6.1 for us
https://travis-ci.org/Workiva/w_common/jobs/611587650#L189

Reply:
ohhh